roles/ceilometer-data/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: install ceilometer compute service
+  upstart_service: name=ceilometer-agent-compute
+                   user=ceilometer
+                   cmd=/usr/local/bin/ceilometer-agent-compute
+  notify:
+    - restart ceilometer services
+
+- name: pip install libvirt-python in venv on compute
+  pip: name=libvirt-python virtualenv={{ ceilometer.virtualenv }}
+  notify:
+    - restart ceilometer services
+  when: openstack_install_method == 'package'
```

Comment:
When installing from source, libvirt-python is installed as a system dependency on the compute node as part of the nova-data role. However, for package install the nova-data role has to install it manually to the nova virtualenv (I mention link & lines below). So for package installs, I also had to install libvirt in my ceilometer virtualenv, since it wasn't installed system-wide.

https://github.com/blueboxgroup/ursula/blob/1c2e57c9b5382813b1f005fa28da814a54f222fd/roles/nova-data/tasks/libvirt.yml (lines 37-48)